.github/workflows/podvm_mkosi.yaml

```diff
@@ -104,32 +190,30 @@ jobs:
           remove-docker-images: "true"
 
       - name: Second checkout
```

Comment:
Checking I understood correctly, it needs to checkout again because of the previous "maximize build space" step?

Anyway, I think we should run the checkout magic for when pull_request_target too.

Reply:
I think so, but that change predates me

> Anyway, I think we should run the checkout magic for when pull_request_target too.

Do you mean do the rebase after this step?

Reply:
yes, rebase after this step if pull_request_target.

Reply:
Done. Thanks